api/src/repositories/sample-location-repository.ts

```diff
@@ -0,0 +1,189 @@
+import { Feature } from 'geojson';
+import SQL from 'sql-template-strings';
+import { z } from 'zod';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { generateGeometryCollectionSQL } from '../utils/spatial-utils';
+import { BaseRepository } from './base-repository';
+import { PostSampleMethod } from './sample-method-repository';
+
+export interface PostSampleLocation {
+  survey_sample_site_id: number | null;
+  survey_id: number;
+  name: string;
+  description: string;
+  survey_sample_site: Feature;
+}
+
+export interface PostSampleLocations {
```

Comment:
Is there any way we could combine these two interfaces into one object in order to avoid confusion? To my eyes, an instance of `PostSampleLocation` could easily be encoded in an instance of `PostSampleLocations`, making the former interface redundant